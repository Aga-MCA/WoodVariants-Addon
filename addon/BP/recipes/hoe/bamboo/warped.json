{
  "format_version": "1.12",
  "minecraft:recipe_shaped": {
    "description": {
      "identifier": "bamboo_warped_hoe"
    },
    "tags": [
      "crafting_table",
      "aga"
    ],
    "pattern": [
      "PP",
      "S",
      "S"
    ],
    "key": {
      "S": {
        "item": "aga_wood_variants:warped_stick"
      },
      "P": {
        "item": "aga_wood_variants:bamboo_panels"
      }
    },
    "result": {
      "item": "aga_wood_variants:bamboo_warped_hoe"
    }
  }
}