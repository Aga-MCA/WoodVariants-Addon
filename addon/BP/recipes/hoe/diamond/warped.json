{
  "format_version": "1.12",
  "minecraft:recipe_shaped": {
    "description": {
      "identifier": "diamond_warped_hoe"
    },
    "tags": [
      "crafting_table",
      "aga"
    ],
    "pattern": [
      "PP",
      "S",
      "S"
    ],
    "key": {
      "S": {
        "item": "aga_wood_variants:warped_stick"
      },
      "P": {
        "item": "minecraft:diamond"
      }
    },
    "result": {
      "item": "aga_wood_variants:diamond_warped_hoe"
    }
  }
}