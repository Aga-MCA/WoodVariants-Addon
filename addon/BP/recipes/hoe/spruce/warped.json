{
  "format_version": "1.12",
  "minecraft:recipe_shaped": {
    "description": {
      "identifier": "spruce_warped_hoe"
    },
    "tags": [
      "crafting_table",
      "aga"
    ],
    "pattern": [
      "PP",
      "S",
      "S"
    ],
    "key": {
      "S": {
        "item": "aga_wood_variants:warped_stick"
      },
      "P": {
        "item": "aga_wood_variants:spruce_panels"
      }
    },
    "result": {
      "item": "aga_wood_variants:spruce_warped_hoe"
    }
  }
}