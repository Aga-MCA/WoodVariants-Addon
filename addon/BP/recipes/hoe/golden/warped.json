{
  "format_version": "1.12",
  "minecraft:recipe_shaped": {
    "description": {
      "identifier": "golden_warped_hoe"
    },
    "tags": [
      "crafting_table",
      "aga"
    ],
    "pattern": [
      "PP",
      "S",
      "S"
    ],
    "key": {
      "S": {
        "item": "aga_wood_variants:warped_stick"
      },
      "P": {
        "item": "minecraft:gold_ingot"
      }
    },
    "result": {
      "item": "aga_wood_variants:golden_warped_hoe"
    }
  }
}